variables.tf

```diff
@@ -54,3 +54,13 @@ variable "adminrouter_grpc_proxy_port" {
   description = ""
   default     = 12379
 }
+
+variable "open_admin_router" {
+  description = "Open admin router to public (80+443 on load balancer). WARNING: attackers could take over your cluster"
```

Comment:
> WARNING: ... **Use a strong password, otherwise** attackers ...

Reply:
well doesn't really count for open. An open cluster might be completely without authentication.